src/applications/personalization/profile/tests/e2e/contact-information/focus-management.cypress.spec.js

```diff
@@ -12,9 +15,9 @@ const setup = () => {
     'v0/profile/ch33_bank_accounts',
     'v0/profile/status',
     'v0/mhv_account',
-    'v0/feature_toggles*',
     'v0/ppiu/payment_information',
   ]);
+  mockFeatureToggles();
```

Comment:
The mockFeatureToggles is needed in various e2e tests now so that the actual feature toggle resolve instead of returning a 500 error as the mockGETEndpoint does.